lib/user/jwt/client.js

```diff
@@ -513,7 +519,11 @@ class Client {
 
       if (logger) logError('API', e)
 
-      client.handleRequestError(e)
+      const { code, message, response } = e
+
+      error({ ...(code ? { code } : {}), ...(message ? { message } : {}) })
```

Comment:
This will not add the `data` key nor the `message` key to the object if the corresponding variable is falsy

Reply:
What does the `...` represent in JS?